client/devdocs/design/playground-scope.js

```diff
@@ -1,123 +1,123 @@
-/** @format */
-
 /**
  * External dependencies
  */
-export Gridicon from 'gridicons';
-export SocialLogo from 'social-logos';
+export { default as Gridicon } from 'gridicons';
```

Comment:
Note: these changes were made to appease ESLint, which doesn't handle `eslint-disable` correctly for parsing errors, such as the one the previous syntax caused.

Reply:
Outdated.